tests/core/pyspec/eth2spec/test/eip7594/networking/test_get_custody_columns.py

```diff
@@ -0,0 +1,49 @@
+import random
+
+from eth2spec.test.context import (
+    single_phase,
+    spec_test,
+    with_eip7594_and_later,
+)
+
+
+def _run_get_custody_columns(spec, rng):
+    node_id = rng.randint(0, 2**32 - 1)
+    custody_subnet_count = rng.randint(0, spec.config.DATA_COLUMN_SIDECAR_SUBNET_COUNT)
```

Comment:
So that it could possibly include the max value:
```suggestion
    custody_subnet_count = rng.randint(0, spec.config.DATA_COLUMN_SIDECAR_SUBNET_COUNT+1)
```

Reply:
I was also confused `randint` with `randrange` all the time. 😄

[randint](https://docs.python.org/3/library/random.html#random.randint):  Return a random integer `N` such that `a <= N <= b`. Alias for `randrange(a, b+1)`.

Reply:
Ah I see. Yes, that is confusing. All good then!